packages/app/index.css

```diff
@@ -0,0 +1,21 @@
+@import './compiledStyle.css';
```

Comment:
I think you deleted this file.

Reply:
yes.. fixed in 4bdb939bf1512fcb307e91af91ffd2fc6a71087f